help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,160 @@
+-   **File Type** - Select the format Experience Platform should use for the exported files. Currently, the only file type Acxiom processing will be expecting is CSV
+
+[!NOTE]
+When selecting the CSV option, Delimiter, Quote Character, Escape Character, Empty Value, Null Value, Compression format, and Include manifest file options will be presented, the following document explains these settings in more detail configure the formatting options.
+
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-connections/media/image8.png)
+
+### Existing Account
+
+Accounts already defined using the Merkury Enterprise Connections destination appear in a list pop-up. When selected, you can see details on the account in the right rail. View the example from the UI, when you navigate to **Destinations** \> **Accounts**;
+
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-connections/media/image5.png)
+
+## Enable alerts
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/alerts).
+
+When you are finished providing details for your destination connection, select **Next**.
+
+## Activate audiences to this destination
+
+[!IMPORTANT]
+- To activate data, you need the View Destinations, Activate Destinations, View Profiles, and View Segments access control permissions. Read the access control overview or contact your product administrator to obtain the required permissions.
+- To export identities, you need the View Identity Graph access control permission.
+
+
+Read [Activate audience data to batch profile export destinations](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations) for instructions on activating audiences to this destination.
+
+## Mapping suggestions
+
+The correct processing of files on the Merkury side requires name and address elements. While not all elements are required, providing as much as possible will aid in successful matching.
+
+Mapping suggestions are provided in the table below listing attributes on your destination side that are used by Merkury processing that customers can map profile attributes to. Treat these elements as suggestions as not all elements are required, and the source values will depend on the needs of the account.
+
+|Target Field|Source Description|
+|---|---|
+|id|Identity field to be used to map merkury data to AEP through the Merkury Enterprise Identity Resolution Source connector|
+|Input_First_Name|The person. name. firstName value in Experience Platform.|
```

Comment:
please no spaces in `person. name. firstName`. suggestion also to wrap these attributes in `` so they stand out on the page

Reply:
Addressed